[🐛 Bug]: Browser type primitive mismatches WebdriverIO.Browser global type

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

Tested in 8.15.10 (latest as of writing), introduced in 8.11.0

### Node.js Version

Verified in 18.17.1, 20.5.1

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
n/a
```


### What happened?

As of PR https://github.com/webdriverio/webdriverio/pull/9640, WebdriverIO's typings include two copies of the `Browser` type, one a global type within the `WebdriverIO` namespace and one a type primitive exported from the module.

PR https://github.com/webdriverio/webdriverio/pull/10447 made an update to to the primitive export, but not the global namespace version.

In practice, this means that code which tries to use `import type { Browser } from 'webdriverio'` is incompatible with webdriverio functions that return `WebdriverIO.Browser`, such as `remote`.

### What is your expected behavior?

The two copies of the type should match, ideally with one defined in terms of the other to avoid this issue in the future

### How to reproduce the bug.

```ts
// repro.ts
import { type Browser, remote } from 'webdriverio';

async function main() {
    let browser: Browser; // works if using WebdriverIO.Browser instead

    browser = await remote({
        capabilities: { browserName: 'irrelevant' }
    });
}

```

```sh
$ npm init -y
$ npm install webdriverio@8.15.10 typescript@5.2.2
$ npx tsc --noEmit --skipLibCheck ./repro.ts
```

### Relevant log output

```typescript
repro.ts:7:5 - error TS2740: Type 'Browser' is missing the following properties from type 'Browser': sessionStatus, sessionNew, sessionEnd, sessionSubscribe, and 25 more.

7     browser = await remote({
      ~~~~~~~


Found 1 error in repro.ts:
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

--- a/packages/webdriverio/src/index.ts
+++ b/packages/webdriverio/src/index.ts
@@ -15,9 +15,7 @@ import { getPrototype, addLocatorStrategyHandler, isStub } from './utils/index.j
 import type { AttachOptions, RemoteOptions } from './types.js'
 import type * as elementCommands from './commands/element.js'
 
-export * from './types.js'
-export * from './utils/interception/types.js'
-
+export { Selector, CustomInstanceCommands } from './types.js'
 export const Key = KeyConstant
 export const SevereServiceError = SevereServiceErrorImport
 

